addons/stock/models/stock_picking.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import json
```

Comment:
put it next to "import time"  or move import time after this one, i don't remember the pep8 rule